deprecate batch (#284)
flashbots/flashbots-docs: docs/flashbots-protect/rpc/ratelimiting.md

If you are a wallet or application integrating with protect and you require a higher rate limit please reach out to [bertcmiller](https://twitter.com/bertcmiller).

## Batch request support

> The Flashbots Protect RPC is removing the ability to make batch json rpc requests.

Reviewer: ```suggestion
The Flashbots Protect RPC is not supporting batch JSON-RPC requests.
```
